File: tmuxer.pl
#!/usr/bin/env perl

use strict;
use warnings;

my $TMUX='/usr/bin/tmux';

my $session_name = substr($ARGV[0],0,index($ARGV[0],'.tconf'));

print "Got Argument: $ARGV[0]\n";
print "Got SessionName: $session_name\n";

open(HF, "$ARGV[0]") ; #Host File

while(<HF>) {
    chomp;
    print "$_\n";
}

close(HF); 

File: tmuxer.rb
#!/usr/bin/env ruby
#

$TMUX = '/usr/bin/tmux'

if not ARGV[0].nil? then   tconfFile = ARGV[0] else print "Need Input .tconf file.\ne.g. : #{$0} my.tconf "; exit 1; end
tmuxSessionName = ARGV[0].split('.tconf')

$hostlist = []

File.readlines(tconfFile).each { |h| $hostlist.push( h.chomp("\n") ) if h !~ /^\n$/  }
print $hostlist
